index.js

```diff
@@ -0,0 +1,217 @@
+    <li class="skil button">${ProjectsMobile[i].Tecnologies[2]}</li>
+  </ul>
+  </div>
+  <button class ="see-project-2" id="seeproject${[i]}" onclick="popupp()">${ProjectsMobile[i].Link}</button>
+  `;
+  cardContainer.innerHTML += projectM;
+}
+
+// Get the popup mobile
+const modal = document.getElementById('myModal');
+modal.innerHTML = `<div class="modal-content">
+<div class="modalHead-close">
+<h2 class = "modalHeader" id="modalHeader">Multi Post Stories</h2>
+ <span class="close">&times;</span>
+ </div>
+ <div>
+  <ul id="modalTech" class ="modalTech">
+      <li>HTML</li>
+      <li>Bootstrap</li>
+      <li>Ruby on rails</li>
+  </ul>
+  <img src="/assets/Portfolio.png" alt="popup image">
+  <p class = "modal-pm" >Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent</p>
+  <div class = "live-source">
+  <a href="https://www.google.com/">see live <img src="/assets/Icon.png" alt="google"></a>
+  <a href="https://github.com/">see source <img src="/assets/Vector.png" alt=""></a>
+  </div>
+</div>
+</div>`;
+const span = document.getElementsByClassName('close')[0];
+function popupp() {
+  modal.style.display = 'block';
+}
+
+function closee() {
+  modal.style.display = 'none';
+}
+span.onclick = function () {
+  closee();
+};
+
+const cardDesk = document.querySelector('#card123');
+const card1 = `<div class="desk-card-pro1 all-cards">
+<button class="desk-see-project-1" id="seeprojectyy" onclick="popupd()">See Project</button>
+</div>`;
+cardDesk.innerHTML += card1;
+const card2 = `<div class="desk-card-d1 all-cards">
+<div class="no-hover">
+<h2 class="desk-prof-art-dash">Data Dashboard Healthcare</h2>
+<p class="desk-p22-d">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+been the industry's standard</p>
+<ul class="desk-skils-d">
+<li class="desk-skil button-d">html</li>
+<li class="desk-skil button-d">bootstrap</li>
+<li class="desk-skil button-d">Ruby</li>
+</ul>
+</div>
+<button class="desk-see-project-1c" onclick="popupd()">See Project</button>
+</div>`;
+cardDesk.innerHTML += card2;
+const card3 = `<div class="desk-card-w1 all-cards">
+<div class="no-hover">
+<h2 class="desk-prof-art-w">Website Protfolio </h2>
+<p class="desk-w">A daily selection of privately personalized reads; no accounts or sign-ups required. has been
+the industry's standard</p>
+<ul class="desk-skils-d">
+<li class="desk-button-d button-d">html</li>
+<li class="desk-skil button-d">bootstrap</li>
+<li class="desk-skil button-d">Ruby</li>
+</ul>
+</div>
+<button class="desk-see-project-1c" onclick="popupd()">See Project</button>
+</div>`;
+cardDesk.innerHTML += card3;
```

Comment:
- [ ] Please make sure to only use the array of project objects to render the project cards as I noticed from the above code you're hardcoding each project card. You've already created an array of project cards so please fix the code from lines `39` to `53` to properly render them. ✔